src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlDocument.java

```diff
@@ -68,16 +67,16 @@ public void write(DocFile docFile) throws DocFileIOException {
     @Override
     public String toString() {
         try (Writer writer = new StringWriter()) {
-            write(writer);
+            write(writer, "\n");
             return writer.toString();
         } catch (IOException e) {
             throw new Error(e);
         }
     }
 
-    private void write(Writer writer) throws IOException {
+    private void write(Writer writer, String newline) throws IOException {
         writer.write(docType.text);
-        writer.write(DocletConstants.NL);
-        docContent.write(writer, true);
+        writer.write(newline);
+        docContent.write(writer, newline,true);
```

Comment:
Missing space after comma

Reply:
Fixed.  
This is a mildly annoying side-effect of IDEA wanting to label boolean arguments: it makes the missing spaces much harder to see.